pkg/kwokctl/runtime/compose/cluster.go

```diff
@@ -46,15 +46,32 @@ import (
 // Cluster is a implementation of Runtime for docker-compose.
 type Cluster struct {
 	*runtime.Cluster
+
+	driver string
```

Comment:
```suggestion
	runtime string
```

Maintain contextually consistent naming